server/account.go

```diff
@@ -0,0 +1,65 @@
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"strconv"
+
+	"chai/database/sqlc"
+)
+
+type accountRes struct {
+	Message string       `json:"message"`
+	Data    sqlc.Account `json:"accounts"`
+}
+
+func (a *App) AccountHandler(w http.ResponseWriter, r *http.Request) {
+	switch r.Method {
+	case http.MethodPost:
+		a.InsertAccountHandler(w, r)
+	case http.MethodGet:
+		a.SelectAccountHandler(w, r)
+	default:
+		http.Error(w, "Method not allowed", http.StatusMethodNotAllowed)
+	}
+}
+
+func (a *App) InsertAccountHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
We may want to keep this and the other handler private, since it's more of an implementation detail than it is a handler that we want to consume when registering the routes.

This is more for the sake of having more relevant autocomplete when registering routes than anything else.

```suggestion
func (a *App) insertAccountHandler(w http.ResponseWriter, r *http.Request) {
```